src/Setup/DevDivVsix/PortableFacades/PortableFacades.swr

```diff
@@ -18,7 +18,6 @@ folder InstallDir:\Common7\IDE\PrivateAssemblies
     file source="$(NuGetPackageRoot)\System.IO.FileSystem.Primitives\4.3.0\lib\net46\System.IO.FileSystem.Primitives.dll" vs.file.ngen=yes
     file source="$(NuGetPackageRoot)\System.IO.Pipes.AccessControl\4.3.0\runtimes\win\lib\net46\System.IO.Pipes.AccessControl.dll" vs.file.ngen=yes
     file source="$(NuGetPackageRoot)\System.IO.Pipes\4.3.0\runtimes\win\lib\net46\System.IO.Pipes.dll" vs.file.ngen=yes
-    file source="$(NuGetPackageRoot)\System.Net.Http\4.3.0\runtimes\win\lib\net46\System.Net.Http.dll" vs.file.ngen=yes
```

Comment:
Should remove System.Diagnostics.DiagnosticSource as well. It's only needed by System.Net.Http

Reply:
Ah, hmm, which then rolls back 2705fd4d9071b8e2ed15662a2e345af875caae1b?

Reply:
Makes sense. @genlu want to take care of it?

Reply:
That's at least my understanding last time I looked through all the nuspec files and got the dependency order.

CC @terrajobst as yet another reminder how impossible this is for us to deal with.

Reply:
Done